### ISSUE TITLE
API v2.0 is GONE (status code 410). Need to find the new endpoint

### ISSUE THREAD

**Issue by gido** [gido/postcardcreator#12]
This lib doesn't work anymore since it look like the post changed there endpoints.

```
$ curl -X GET https://postcardcreator.post.ch/rest/2.0/users/current \
  -H 'accept: */*' \
  -H 'authorization: Bearer MySecretToken ' \
  -H 'content-type: application/json' \
  -sv

< HTTP/1.1 410 Gone
< Server: nginx/1.11.10
< Date: Mon, 09 Jul 2018 18:23:54 GMT
< Content-Length: 0
< Connection: keep-alive
< Keep-Alive: timeout=5
```

**Comment by gido** [comment]
Ok, found it: `https://postcardcreator.post.ch/rest/2.2/`

**Comment by gido** [comment]
closed with the release of 2.2 ed5aba9c53099f522ae8260b120767cb4131629e
